Enrise/MM-swagger-ui
# Feature/add latest master

### Context
Merge latest Swagger UI `master`. Since it was a while since I worked on SwaggerUI, I didn't know how to serve swagger locally.

### What has been done?
- Merge latest `master` of SwaggerUI
- Update README: include a section about how to serve files locally
- Remove `deploy.sh` in Git by adding it to `.gitignore`

### How to test
See https://github.com/Enrise/Terberg/pull/812, or serve locally via the instructions found in `README.md`
